network/Message.h

```diff
@@ -72,7 +72,7 @@ class FO_COMMON_API Message {
         LOBBY_EXIT,             ///< sent to server by clients when a player leaves the multiplayer lobby, or by server to clients when a player leaves the multiplayer lobby
         START_MP_GAME,          ///< sent to server (by the "host" client only) when the settings in the MP lobby are satisfactory and it is time to start the game
         SAVE_GAME_INITIATE,     ///< sent to server (by the "host" client only) when a game is to be saved
-        SAVE_GAME_DATA_REQUEST, ///< sent to clients by the server when the game is being saved and the server needs save state info from clients
+        SAVE_GAME_DATA_REQUEST, ///< sent to clients by the server when the game is being saved and the server needs save state info from clients (removed)
```

Comment:
why not actually removed?

Reply:
Also, isn't this still useful when doing a manual save in multiplayer?

Reply:
If you remove it it will shift numbers of other messages. If it is OK I could remove it.
It should work without server-side request in multiplayer too else it cause #2275

Reply:
I don't think the numbers need to be consistent if the change breaks compatibility anyway.

I'm not saying that there should be a separate save data request after sending turn orders in multiplayer. I'm saying that it should still be possible to manually initiate a save in multiplayer in the middle of a turn, which presumably will require the server to request save state info from the human players.

Reply:
Good. Wonder if it possible to reset boost serialization and cleanup deprecated code.

Server stores save state between turn in `ServerApp::m_turn_sequence` and updates it on player sends turn orders so it doesn't need to request it by itself.

Reply:
*in the middle of a turn* ... ie. after some orders have been issued, but before the turn is ended by all players.

Reply:
The game has an `UNREADY` message which simply marks empire unready and doesn't clear orders set sent before so them could be saved. May be better to split readiness and committing orders?

Reply:
That doesn't seem relevant. The point is what happens if a save is initiated while a player is still playing their turn. It should save the orders they have issued so far at that time. It's irrelevant what happens if they are "ready" or have become unready again after being ready.

Reply:
It looks like turn order design issue so I've opened thread: https://freeorion.org/forum/viewtopic.php?f=6&t=11198